homeassistant/components/wallbox/const.py

```diff
@@ -0,0 +1,99 @@
+"""Constants for the Wallbox integration."""
+from homeassistant.const import (
+    CONF_ICON,
+    CONF_NAME,
+    CONF_UNIT_OF_MEASUREMENT,
+    ELECTRICAL_CURRENT_AMPERE,
+    ENERGY_KILO_WATT_HOUR,
+    LENGTH_KILOMETERS,
+    PERCENTAGE,
+    POWER_KILO_WATT,
+    STATE_UNAVAILABLE,
+)
+
+DOMAIN = "wallbox"
+
+CONF_STATION = "station"
+
+CONF_CONNECTIONS = "connections"
+CONF_ROUND = "round"
+
+CONF_SENSOR_TYPES = {
+    "charging_power": {
+        CONF_ICON: "mdi:ev-station",
```

Comment:
Please set device class for as many sensor types as possible. When setting device class for an entity we normally don't set a custom icon.

https://developers.home-assistant.io/docs/core/entity/sensor#available-device-classes